Breadcrumbs.py

```diff
@@ -189,39 +156,28 @@ def on_selection_modified_async(self, view):
     settings = sublime.load_settings('Breadcrumbs.sublime-settings')
     if settings.get('breadcrumbs_statusbar', True):
       separator = settings.get('breadcrumbs_separator', u' › ')
-      view.set_status('breadcrumbs', separator.join(make_breadcrumbs(view) + ['']))
+      view.set_status('breadcrumbs', separator.join(make_breadcrumbs(view) + ['']).rstrip(separator))
```

Comment:
Right... duh.. That's just me being hasty. If you think this is a good direction I will go through it all and clean up.